### PR TITLE
Fix code sample formatting

### PR DESCRIPTION
**Pull request by ChrisMaddock**
The formatting around this code sample was previously both rendering incorrectly on the website only, and numbering incorrectly on the github wiki (only).

Hopefully this will render and number correctly on both sites! May be worth checking how it comes out on the site, as I'm not sure.

https://chocolatey.org/docs/automatic-packages#setup-1

Thanks guys. :smile:

### CODE REVIEW

**Comment by ChrisMaddock** [comment]
(The &cacute; was also rendering incorrectly on the website - hopefully that will fix it!)

File: AutomaticPackages.md [hunk 1]
```diff
@@ -60,10 +60,12 @@ This tool makes use of [Ketarin](https://chocolatey.org/packages/ketarin). Ketar
 1. Check the config in `$env:ChocolateyInstall\lib\ChocolateyPackageUpdater\tools\chocopkgup\chocopkgup.exe.config`. The `PackagesFolder` key should point to where your repository is located.
 1. Create a scheduled task (in Windows). This is the command (edit the path to `cmd.exe` accordingly): `C:\Windows\System32\cmd.exe /c c:\tools\chocolateypackageupdater\ketarinupdate.cmd`
 1. Alternatively to stop the command window from opening on Windows, you can create a VBS script as well and put the path to the `.vbs` file instead of `ketarinupdate.cmd` as the command to run. The file should have the following:
-~~~vb
-Set objShell = WScript.CreateObject("WScript.Shell")
-objShell.Run("C:\tools\ChocolateyPackageUpdater\ketarinupdate.cmd"), 0, True
-~~~
+
+    ```vb
```
Comment (ferventcoder) [inline AutomaticPackages.md]: Let's still use `~~~` - the extra line before the start is what would have fixed it, so good job here.

Comment (ChrisMaddock) [inline AutomaticPackages.md]: Sorry - I switched it to match the same style as the other code sample in this file. 😄 I've changed both to be `~~~` now, if that's the convention.

**Comment by ChrisMaddock** [comment]
These are all in separate commits as I'm doing this through the web UI, which I don't think I can use to squash. Are you ok to do that on commit? I can pull the repo down and do this properly if that's easier! 😄

File: AutomaticPackages.md (hunk 1 above)
Comment (ferventcoder) [inline AutomaticPackages.md]: No worries, older document.

**Comment by ferventcoder** [comment]
> These are all in separate commits as I'm doing this through the web UI, which I don't think I can use to squash.

For the wiki documentation, I'm less concerned about pristine commits. Choco source code is a totally different story.

**Comment by ferventcoder** [comment]
Or in other words, no need. 😄

**Comment by ferventcoder** [comment]
Thanks!

**Pull request merged**